lib/smart_answer_flows/coronavirus-employee-risk-assessment/outcomes/go_back_to_work.govspeak.erb

```diff
@@ -1,9 +1,56 @@
+<% render_content_for :title do %>
+  You can go back to work
+<% end %>
+
 <% render_content_for :body do %>
-  $CTA
-    ## You can go back to work
-    You can go back into work, but your employer must make arrangements for you to work safely.
 
-    [Check the guidance on working safely, and what you will need to do](https://www.gov.uk/coronavirus-business-reopening)
-  $CTA
+  <% if calculator.are_you_vulnerable == "yes" %>
+    $CTA
+      ## If you are clinically vulnerable: get your employer to make arrangements for you to work safely
+
+      Clinically vulnerable people are at higher risk of severe illness (for example, people with pre-existing conditions). Therefore, you should take extra care in observing social distancing. Your employer should help you to work from home, either in your current role or in an alternative role.
+
+      If you are clinically vulnerable (but not extremely clinically vulnerable) and cannot work from home, your employer should offer you the option of the safest available on-site roles, enabling you to stay 2 metres away from others.
+
+      If you have to spend time within 2 metres of others, employers should carefully assess whether this involves an acceptable level of risk. As for any workplace risk, employers must take into account specific duties to those with protected characteristics, including, for example, pregnant women who are entitled to a suspension on full pay if suitable roles cannot be found.
+
+      Find out more about how to work safely in your workplace.
+    $CTA
+  <% end %>
+
+  <% if calculator.do_you_live_with_someone_vulnerable == "yes" %>
+    $CTA
+      ## If you live with someone extremely clinically vulnerable: get advice on how to socially distance at work and keep your household safe
+
+      Employers should pay particular attention to people who live with clinically extremely vulnerable individuals.
+
+      You and your household should follow the guidance on how to protect clinically extremely vulnerable people.
+    $CTA
+  <% end %>
+
+  <% if calculator.have_childcare_responsibility == "yes" %>
+    $CTA
+      ## If you need help with childcare or dependants: Find out if you can send your children back to school or take time off
+
+      ### Schools and childcare
+      Schools in England may open for: primary schools for children in reception, year 1 and year 6 nurseries and other early years settings, including childminders children of key workers and vulnerable children. Find out if you are classed as a key worker, and if you can send your children back to school.
```

Comment:
This needs some formatting love to match the figma prototype ++ it is missing links.


**What it should look like** 
<img width="837" alt="Screenshot 2020-06-25 at 10 19 34" src="https://user-images.githubusercontent.com/7116819/85692616-7672b000-b6cd-11ea-91b2-c1fd0cfa814e.png">

**What it currently looks like**
<img width="906" alt="Screenshot 2020-06-25 at 10 19 50" src="https://user-images.githubusercontent.com/7116819/85692655-7d99be00-b6cd-11ea-91ea-c03a5288a208.png">